src/net.cpp

```diff
@@ -45,6 +45,8 @@
 
 // We add a random period time (0 to 1 seconds) to feeler connections to prevent synchronization.
 #define FEELER_SLEEP_WINDOW 1
+/** Run the feeler connection loop once every 2 minutes or 120 seconds. **/
+static const int FEELER_INTERVAL = 60;
```

Comment:
@EthanHeilman My next commit will remove FEELER_SLEEP_WINDOW as it's redundant. I.e.  RAND(60)+RAND(1) = RAND(61)... i.e. we don't need two random numbers.